src/Function/Properties.agda

```diff
@@ -0,0 +1,25 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some generic function properties.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Function.Properties where
```

Comment:
As discussed in #759 please don't add these in until we sort out how we're reorganising these in relation to the current existing hierarchy. Until then please write out the type in full.